aws/table_ aws_ecrpublic_repository.go

```diff
@@ -0,0 +1,290 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ecrpublic"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEcrpublicRepository(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ecrpublic_repository",
+		Description: "AWS ECRPublic Repository",
```

Comment:
https://docs.aws.amazon.com/AmazonECR/latest/public/public-repositories.html
```suggestion
		Description: "AWS ECR Public Repository",
```